src/App.tsx

```diff
@@ -1,8 +1,26 @@
-import React from 'react';
-import './App.css';
+import { useState, useEffect } from 'react';
+import './App.scss';
+import Search from './components/Search/index';
+import searchCities from './api/searchCities';
 
-function App() {
-    return <div className="App" />;
+function App(): JSX.Element {
+    const [search, setSearch] = useState<string>('');
+    const [cities, setCities] = useState<string[]>([]);
+    useEffect(() => {
+        if (search) {
+            searchCities(search).then((res) => {
+                setCities(res);
+            });
+        }
+    }, [search]);
+    function searchInputChanging(_: React.SyntheticEvent, value: string) {
```

Comment:
again bad naming. this is kind of event, so it should have `on` prefix and next should an action (search in this case). so rename to `onSearch` or `onCitiesSearch`